torch/nn/quantized/_qscheme.py

```diff
@@ -0,0 +1,37 @@
+from ..._jit_internal import weak_script
+
+# NB: Keep this file in sync with enums in c10/core/QScheme.h
+
+
+@weak_script
+def get_enum(qscheme):
+    # type: (str) -> int
+    if qscheme == 'none':
```

Comment:
why do you need none?  Can you just use an optional qscheme / python None for this case?